docs/workflows.rst

```diff
@@ -3,24 +3,24 @@
 Translation workflows
 =====================
 
-Using Weblate is a process that brings your users closer to you, by bringing
-you closer to your translators.  It is up to you to decide how many of its
+Using Weblate is a process bringing your users closer to you, where you are
```

Comment:
This sentence doesn't really explain anything.

Reply:
If you know that the sentence is not useful, feel free to overwrite it or remove it next time.